### ISSUE TITLE
the link of cornerstoneWADOImageLoader.js is the same of cornerstoneWADOImageLoader.bundle.min.js

### ISSUE THREAD

**Issue by chenzongxi** [cornerstonejs/cornerstoneWADOImageLoader#447]
Get the distributed unminimized file:

[cornerstoneWADOImageLoader.js](https://unpkg.com/cornerstone-wado-image-loader)
or the distributed minimized file:

[cornerstoneWADOImageLoader.bundle.min.js](https://unpkg.com/cornerstone-wado-image-loader)

so where can find the file of cornerstoneWADOImageLoader.js?
thanks

**Comment by ahmetaltay33** [comment]
Yes, I could not find unminimized version of cornerstoneWADOImageLoader either. I tried recompile from the source code but does not work.

**Comment by keyhooon** [comment]
Yes, I could not find too. I need it for debugging operation.
